### PR TITLE
Add stylelint

### PR DESCRIPTION
**Pull request by YayunHuang**
## Blocked by 
- #110 

## What's in this PR?
Ref: MUP-158

### CODE REVIEW

**Comment by pkong-ds** [comment]
Please rebase

File: package.json [hunk 1]
```diff
@@ -29,7 +30,11 @@
     "eslint": "^8.46.0",
     "eslint-plugin-astro": "^0.27.2",
     "eslint-plugin-jsx-a11y": "^6.7.1",
+    "postcss-html": "^1.7.0",
     "prettier": "^3.0.1",
-    "prettier-plugin-astro": "^0.11.0"
+    "prettier-plugin-astro": "^0.11.0",
+    "stylelint": "^16.8.2",
+    "stylelint-config-html": "^1.1.0",
+    "stylelint-config-standard": "^36.0.1"
```
Comment (pkong-ds) [inline package.json:38]: curious: Which guide did you follow to install 5 packages for stylelint?

**Review by pkong-ds — COMMENTED** [review]
Thanks a lot :sadfrog: I did not expect to lint all files in this PR, was planning to open another issue and follow up (like js->ts)

Yayun 👍

File: package.json (hunk 1 above)
Comment (YayunHuang) [reply package.json:38]: [Astro doc](https://docs.astro.build/en/editor-setup/#stylelint) says can follow https://github.com/ota-meshi/stylelint-config-html
and `create-stylelint` just use for init the config

**Comment by YayunHuang** [comment]
@pkong-ds sure, haven't add `.ts` in `lint_astro` for now, will add it in another pr, I only fix errors in `.css` files and `<style>` in `.astro` files

**Comment by pkong-ds** [comment]
> will add it in another pr

I think fix all lint errors in one go is very tedious. I rmb @cychiuae instruction was: ignore all files first, when we work on that file, fix errors in that file.

That way we can achieve incremental improvement.

Hence, I suggest you to ignore all files first

**Review by pkong-ds — APPROVED** [review]
LGTM, will merge

**Pull request merged**